src/app/shared/navbar/searchbar/searchbar.html

```diff
@@ -0,0 +1,16 @@
+<div class="search-input-container">
+  <input
+    placeholder="Search"
+    type="text"
+    (focus)="toggleIsExpanded(true)"
+    (blur)="toggleIsExpanded(false)"
+    (keyup.enter)="navigate($event.target.value.toLowerCase())"
```

Comment:
Does this also navigate when you click on the option?Also have you considered switching to the `optionSelections` observable on the trigger rather than this event? It emits any option object that is selected (via the keyboard or the mouse).